samples/subsys/usb/cdc_acm/frdm_k64f.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2019, Phytec Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* -DDTC_OVERLAY_FILE=dap_hid_frdm_k64f.overlay */
```

Comment:
Is this old comment when your overlay had this name? With default name ${BOARD}.overlay it will be taken into account for all builds for this sample for the ${BOARD}.

I think we need to create another special sample.